### ISSUE TITLE
Adjust logo size

### ISSUE THREAD

**Issue by luke-** [humhub/humhub-modules-enterprise-theme#17]
In the `enterprise-theme\layouts\main.php` the size of the max converted logo is too low.

We should choose a higher value here, because on the one hand the width of the sidebar can be changed and on the other hand HighRes displays may otherwise show logos in a pixelated way.  

We should make sure that the maximum size is defined via stylesheet.

**Comment by yurabakhtin** [comment]
@luke- As I understand currently there is a code:

```
<img id="img-logo" class="img-rounded"
    src="<?= LogoImage::getUrl(400, 200); ?>"
```

and the logo image width is limited by css:

```
@ee-sidebar-width: 250px;
...
#img-logo {
  margin-left: 15px;
  max-width: @ee-sidebar-width - 40px;
}
```

and the less var `@ee-sidebar-width` may be changed in another theme.
So we should use a higher size from PHP side, right? but what size should we use there, 1000x1000 is enough? I think yes, because I am not sure sidebar may be so wider more 1000px.

**Comment by luke-** [comment]
Ok perfect, that looks good. I think the restriction via stylesheets is quite new and that's why the generated image size was still as small as possible, just to be on the safe side.  

Since the standard sidebar has a width of 250px, a generated 600x600px should easily suffice.

**Comment by yurabakhtin** [comment]
@luke- ok, MR https://git.humhub.org/professional/enterprise-theme/-/merge_requests/2

**Comment by luke-** [comment]
Great!

Will be fixed with enterprise-theme version 1.4.4
